### PR TITLE
I added the RVM description to the doc signedoff-by-Amarachi-Azubuike

### PR DESCRIPTION
**Pull request by emmanuellar**

### CODE REVIEW

**Comment by Jashpatel1** [comment]
The DCO checks are failing as you haven't signed your commits. Can you please have a look into [signing instructions.](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) This would definitely help you pass the DCO check.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
 
 <a href="./assets/images/git-signoff-vscode.png" ><img src="./assets/images/git-signoff-vscode.png" width="50%"><a>
 
+### Set Up Your Development Environment
+Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also install Ruby Version Manager [RVM](https://rvm.io) - a command-line tool which allows you to easily install, manage and work with multiple ruby environments on your local machine. Alternatively, if you're running Windows 10, updated to version 1903 Build 18362 or higher you can upgrade to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:65]: Alternatively, if you're running Windows 10, update to version 1903 or higher. You can also shift to Windows Subsystem for Linux.....(same as previous)

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (emmanuellar) [reply CONTRIBUTING.md:65]: Okay. Will effect changes

**Comment by leecalcote** [comment]
> The DCO checks are failing as you haven't signed your commits. Can you please have a look into [signing instructions.](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) This would definitely help you pass the DCO check.

@Jashpatel1 excellent of you to point these out. 👍 

For this immediate PR / set of commits, it's good to highlight https://github.com/layer5io/layer5/pull/682/checks?check_run_id=1034135524 as instructions to provide an immediate remedy (on this PR only).

File: CONTRIBUTING.md (hunk 1 above)
Comment (emmanuellar) [reply CONTRIBUTING.md:65]: I've effected these changes @Nikhil-Ladha

**Comment by emmanuellar** [comment]
I just made a new commit where I signed off, kindly confirm if it worked. Thank you

**Comment by leecalcote** [comment]
Yay! Nice work, @emmanuellar. 🎈

**Comment by emmanuellar** [comment]
Yaaayyyy💃🏽💃🏽💃🏽

Thank you for being so patient.

**Comment by PluckyPrecious** [comment]
You did it @emmanuellar !🥳
I just want to mention that it’s great to provide a description of work done when making a PR, and also mention the issue fixed.😊
> This PR fixes #676

**Pull request merged**